packages/analytics-core/src/diagnostics/diagnostic.ts

```diff
@@ -0,0 +1,71 @@
+import { Diagnostic, DiagnosticOptions } from '@amplitude/analytics-types';
+import { DIAGNOSTIC_ENDPOINT } from '../diagnostics/constants';
+import { DiagnosticOmniMetrics } from './typings';
+import { DIAGNOSTIC_METADATA_TYPE } from './constants';
+
+export class BaseDiagnostic implements Diagnostic {
+  isDisabled = false;
```

Comment:
Since this is going to be added automatically in Node and ReactNative can we have the BaseDiagnostic be disabled by default, and then have BrowserDiagnostic enabled by default?

Reply:
Disable BaseDiagnostic in https://github.com/amplitude/Amplitude-TypeScript/pull/568/commits/80a811483b2725fc81124966fb664c6640f5c37c